mod-record-specifications-validator/src/main/java/org/folio/rspec/validation/validator/marc/impl/MarcFieldNonRepeatable1xxFieldRuleValidator.java

```diff
@@ -2,33 +2,27 @@
 
 import java.util.List;
 import java.util.Map;
-import org.folio.rspec.domain.dto.DefinitionType;
-import org.folio.rspec.domain.dto.SeverityType;
+import org.folio.rspec.domain.dto.FamilyProfile;
 import org.folio.rspec.domain.dto.SpecificationDto;
 import org.folio.rspec.domain.dto.ValidationError;
 import org.folio.rspec.i18n.TranslationProvider;
 import org.folio.rspec.validation.validator.SpecificationRuleCode;
-import org.folio.rspec.validation.validator.SpecificationRuleValidator;
 import org.folio.rspec.validation.validator.marc.model.MarcField;
 import org.folio.rspec.validation.validator.marc.model.MarcRuleCode;
-import org.folio.rspec.validation.validator.marc.utils.TagsMatcher;
 
 class MarcFieldNonRepeatable1xxFieldRuleValidator
-  implements SpecificationRuleValidator<Map<String, List<MarcField>>, SpecificationDto> {
-
-  private final TranslationProvider translationProvider;
+  extends Abstract1xxFieldRuleValidator {
 
   MarcFieldNonRepeatable1xxFieldRuleValidator(TranslationProvider translationProvider) {
-    this.translationProvider = translationProvider;
+    super(translationProvider);
   }
 
   @Override
   public List<ValidationError> validate(Map<String, List<MarcField>> fields, SpecificationDto specification) {
-    List<MarcField> all1xxFields = fields.keySet().stream()
-      .filter(TagsMatcher::matches1xx)
-      .flatMap(tag -> fields.get(tag).stream())
-      .toList();
-
+    if (!FamilyProfile.BIBLIOGRAPHIC.equals(specification.getProfile())) {
+      return List.of();
+    }
```

Comment:
Please do not rely on family profile. Rules are not specific to any profile.

Reply:
Oh yeah, fixed.